tox.ini

```diff
@@ -1,29 +1,22 @@
 [tox]
-envlist = coverage-clean, py27, py33, py34, py35, pypy, coverage-report
+envlist = flake8, py27, py34, py35, py36, py37, pypy, pypy3, coverage
 
 [testenv]
+usedevelop = true
```

Comment:
I would prefer not to have this by default.

Reply:
I'd like to keep it for the reasons as described in zopefoundation/z3c.layer.pagelet#1.